modules/core/perf/perf_arithm.cpp

```diff
@@ -1,6 +1,6 @@
 #include "perf_precomp.hpp"
-#include "opencv2/core/softfloat.hpp"
 #include <numeric>
+#include "opencv2/core/softfloat.hpp"
```

Comment:
Looks like unnecessary changes in this file.
As there are dedicated PRs to 4.x and 5.x - all questions to the author why do we have these differences.